File: pyjs/pyjamas-0.1/pyjs/tests/test017.js
function TimeSlot() {
    return new __TimeSlot();
}
function __TimeSlot() {
}
__TimeSlot.DAYS = __TimeSlot.prototype.DAYS = new pyjslib_List(["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"]);
__TimeSlot.prototype.getDescription = function() {
    return this.DAYS.__getitem__(this.dayOfWeek) + " " + this.getHrsMins(this.startMinutes) + "-" + this.getHrsMins(this.endMinutes);
};
__TimeSlot.prototype.getHrsMins = function(mins) {
};

File: pyjs/pyjamas-0.1/pyjs/tests/test017.py
class TimeSlot:

    DAYS = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"]

    def getDescription(self):
        return self.DAYS[self.dayOfWeek] + " " + self.getHrsMins(self.startMinutes) + "-" + self.getHrsMins(self.endMinutes)

    def getHrsMins(self, mins):
        pass